internal/controller/operatorconfigmap_controller.go

```diff
@@ -687,6 +713,23 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI(storageClients *v1al
 		}); err != nil {
 			return fmt.Errorf("failed to reconcile nfs driver: %v", err)
 		}
+	} else {
+		nfsDriver := &csiopv1.Driver{}
+		nfsDriver.Name = templates.NfsDriverName
+		nfsDriver.Namespace = c.OperatorNamespace
```

Comment:
We can move this from here and the `if` condition above, to the outside block in someway like this:
```
nfsDriver := &csiopv1.Driver{
    Name: templates.NfsDriverName,
    Namespace: c.OperatorNamespace
}
```